src/main/scala/sbtprotoc/ProtocPlugin.scala

```diff
@@ -192,11 +235,27 @@ object ProtocPlugin extends AutoPlugin {
           } else PB.protocVersion.value
         ("com.google.protobuf" % "protoc" % version) asProtocBinary (),
       },
-      PB.deleteTargetDirectory := true
+      PB.protocExecutable := {
```

Comment:
I moved this up from the config scope to the project one, as:
* protocDependency was only defined in the config scope
* I don't see why one would like to have different `protoc` versions/executable between configurations of the same module?